opendihu_examples/isotonic_contraction/optimize_length_before_prestretch/cuboid_muscle/README.md

```diff
@@ -0,0 +1,17 @@
+# Bayesian Optimization for a cuboid muscle model
+
+## Setup
+- A dummy cuboid muscle geometry. 
+- The solvers for both stretching and contraction are coupled mechanics solver and fastmonodomain solver. In the prestretch process we set dynamic to `False` and add boundary conditions that simulate the muscle being fixed at one side and being pulled at from the other side until it has reached a given length. In the contraction process we set dynamic to `True` and let one end of the muscle free.
+- It uses the electrophysiology CellML model "hodgkin_huxley-razumova" and the incompressible mechanics model "Mooney-Rivlin".
+- No preCICE involved. 
+
+## How to run
+To run a single simulation of stretching and contracting a muscle, go to build_release and run:
+```
+./muscle_contraction_with_prestretch ../settings_contraction_with_prestretch.py 12.0 14.0
```

Comment:
Looks good that you have two arguments. Maybe it would be better it to have a name, e.g.,
```suggestion
./muscle_contraction_with_prestretch ../settings_contraction_with_prestretch.py --stress-free12.0 --after-prestretch14.0
```